openhands/microagent/__init__.py

```diff
@@ -0,0 +1,190 @@
+import io
+from enum import Enum
+from pathlib import Path
+from typing import Dict, Tuple, Union
+
+import frontmatter
+from pydantic import BaseModel, Field
+
+from openhands.core.exceptions import (
+    MicroAgentValidationError,
+)
+
+
+class MicroAgentType(str, Enum):
```

Comment:
I think in general it'd be better to not include any actual content in the `__init__.py` file as it's a bit unexpected. Can we break this off to other files?

Reply:
Done!